.github/workflows/ci-backend.yml

```
@@ -80,13 +80,12 @@ jobs:
               run: sudo apt-get install libxml2-dev libxmlsec1-dev libxmlsec1-openssl
 
             - uses: syphar/restore-virtualenv@v1.2
-              id: cache-backend-tests-saml
+              id: cache-backend-tests
```

For this to be effective I think you'll also need to do this

```suggestion
              id: cache-backend-tests
              with:
                  custom_cache_key_element: v1-
```